pom.xml

```diff
@@ -313,7 +313,8 @@
         <version.com.ibm.async.asyncutil>0.1.0</version.com.ibm.async.asyncutil>
         <version.com.microsoft.azure>8.6.6</version.com.microsoft.azure>
         <version.com.nimbus.jose-jwt>9.23</version.com.nimbus.jose-jwt>
-        <version.com.squareup.okhttp3>3.14.9</version.com.squareup.okhttp3>
+        <legacy.version.com.squareup.okhttp3>3.14.9</legacy.version.com.squareup.okhttp3>
+        <version.com.squareup.okhttp3>4.9.2</version.com.squareup.okhttp3>
```

Comment:
Can we rename this property to "version.com.squareup.okhttp"?

Reply:
That shouldn't be a problem, but that means "okhttp" points to v3 and "okhttp3" points to v4, which is kinda confusing. Not sure why that property was named that way originally, but, when in Rome... :)

Reply:
(Deleting earlier comment)

Can I read this mean "okhttp3" -> "okhttp", and leaving the "legacy" variant (though renamed as well)?

Reply:
I would use "version.com.squareup.okhttp" and "legacy.version.com.squareup.okhttp"

Reply:
OK. In progress now...